app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -125,8 +140,14 @@ protected boolean hasPositiveButton() {
     protected void onPositiveClick() {
```

Comment:
doin just short validation and sending. i can split it to an external validation method